### PR TITLE
support rollout trait

### PR DESCRIPTION
**Pull request by wonderflow**
Signed-off-by: 天元 <jianbo.sjb@alibaba-inc.com>

### CODE REVIEW

File: charts/vela-core/templates/velaConfig.yaml [hunk 1]
```diff
@@ -14,11 +14,11 @@ data:
     }
   flagger.app: |
     {
-      "repo": "flagger",
-      "urL": "https://flagger.app",
+      "repo": "oam-flagger",
+      "urL": "https://oam.dev/flagger/archives/",
```
Comment (zzxwill) [inline charts/vela-core/templates/velaConfig.yaml:18]: We moved to the official repo?

**Review by zzxwill — APPROVED** [review]
/lgtm

File: docs/_sidebar.md [hunk 2]
```diff
@@ -50,9 +50,11 @@
     - [vela workloads](/en/cli/vela_workloads.md)
   - Traits
     - [vela traits](/en/cli/vela_traits.md)
-    - [vela manualscale](/en/cli/vela_manualscaler.md)
-    - [vela scale](/en/cli/vela_scale.md)
+    - [vela scaler](/en/cli/vela_scaler.md)
```
Comment (zzxwill) [inline docs/_sidebar.md:53]: `vela scale` is better. I changed it to a verb.

Comment (wonderflow) [reply docs/_sidebar.md:53]: sure， you could raise a PR for that

**Pull request merged**